.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+
+jobs:
+  build_only:
+    docker: &docker-template
```

Comment:
I have no idea what this syntax means. What's it doing?

Reply:
It's a YAML anchor -- it provides a name which refers to the content of the current key and which can be used to "copy" that content into another key later on.